src/api/app/models/concerns/multibuild_package.rb

```diff
@@ -7,6 +7,8 @@ def valid_multibuild_name?(name)
     end
 
     def striping_multibuild_suffix(name)
+      return name if name.blank?
```

Comment:
This method is supposed to give back a package name in *any* case. The bug must be in the calling code, with this change the problem would just manifest itself somewhere down the call path.

Reply:
There are a couple of calls to this method coming from different files. I'll close this PR for now.